client/components/IgnoreCancelSaveModal.jsx

```diff
@@ -103,30 +108,41 @@ export class IgnoreCancelSaveModalComponent extends React.Component {
         );
     }
 
+    getOkText() {
+        const {item, onGoTo, onSaveAndPost} = this.props.modalProps || {};
+        let okText;
+
+        if (onGoTo) {
+            okText = gettext('GoTo');
```

Comment:
Need a space between, 'Go' and 'To' when used in gettext() 😃

Reply:
Changed it to 'Go-To' as this is the wording used in Authoring